### ISSUE TITLE
`Cell` class has obsolete `kwargs`

### ISSUE THREAD

**Issue by Helveg** [LFPy/LFPy#249]
From reading this source code it looks like there's never a valid use case for there to ever be any `**kwargs` being passed into the Cell class. If this is so why not remove all this code and remove `**kwargs` from `__init__`?

```python
        for key in ['timeres_NEURON', 'timeres_python']:
            if key in kwargs.keys():
                raise DeprecationWarning('cell parameter {} is deprecated. Use dt=float instead'.format(key))
        if 'tstartms' in kwargs.keys():
            raise DeprecationWarning('cell parameter tstartms is deprecated. Use tstart=float instead')
        if 'tstopms' in kwargs.keys():
            raise DeprecationWarning('cell parameter tstopms is deprecated. Use tstop=float instead')
        if 'rm' in kwargs.keys():
            raise DeprecationWarning('Cell parameter rm is deprecated, set parameter passive_parameters=dict(g_pas=1/rm, e_pas=e_pas) instead')
        if 'e_pas' in kwargs.keys():
            raise DeprecationWarning('Cell parameter e_pas is deprecated, set parameter passive_parameters=dict(g_pas=1/rm, e_pas=e_pas) instead')

        # check if there are un-used keyword arguments present in kwargs
        for key, value in kwargs.items():
            raise ValueError(f'The keyword and argument {key}={value} is not valid input to class LFPy.Cell')
```

This is all information that can be conveyed more correctly to the user through **semantic versioning** of your software (introducing breaking changes like changing parameter names only in major version changes) and by keeping your documentation up to date.

**Comment by espenhgn** [comment]
Thanks for the heads up. It's fixed in the 2.2.dev0 branch though.

**Comment by Helveg** [comment]
Sorry, I was looking at master :)
